dartsim/src/AddedMassFeatures_TEST.cc

```diff
@@ -19,7 +19,7 @@
 #include <dart/dynamics/BodyNode.hpp>
 
 #include <gz/common/Console.hh>
-#include <gz/math/eigen3.hh>
+#include <gz/math/eigen3/Conversions.hh>
```

Comment:
this is more precise, so we'll keep it, but I did fix gz-math to properly export eigen3.hh in https://github.com/gazebosim/gz-math/pull/683, which should make some of the gz-sim bazel work easier

Reply:
ah yes thanks for the fix. I incorporated those changes in a gz-math8 patch on BCR: https://github.com/bazelbuild/bazel-central-registry/pull/4875